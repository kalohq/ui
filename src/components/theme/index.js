/* @flow */
export {default} from './theme';
