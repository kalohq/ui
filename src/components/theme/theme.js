/* @flow */

import tokens from '../../design-tokens/tokens.theme.js';

/**
 * ThemeProvider Kalo Theme
 * 
 * This is the core CSS-in-JS theme that is passed in to the
 * global ThemeProvider. It is mainly composed of our core
 * design tokens.
 */

const theme = {
  space: [0, 8, 16, 32, 64],
  breakpoints: [],
  ...tokens,
};

export default theme;
