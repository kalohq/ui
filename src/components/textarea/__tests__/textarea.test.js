/* @flow */
/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import {ThemeProvider} from 'emotion-theming';

import theme from '../../theme';
import Textarea from '../textarea';

describe('Input', () => {
  const defaultProps = {
    size: 'medium',
    theme: 'default',
  };

  test('should render shallow component ok', () => {
    const element = renderer
      .create(
        <ThemeProvider theme={theme}>
          <Textarea {...defaultProps} />
        </ThemeProvider>
      )
      .toJSON();
    expect(element).toMatchSnapshot();
  });
});
