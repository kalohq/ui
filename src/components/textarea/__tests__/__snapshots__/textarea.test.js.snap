// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input should render shallow component ok 1`] = `
<textarea
  className="ui-textarea ui-textarea--animate e1q9f2xp6 css-0"
  disabled={undefined}
  onChange={undefined}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onScroll={[Function]}
  readOnly={undefined}
  size="medium"
  style={
    Object {
      "height": undefined,
    }
  }
/>
`;
